src/common/translations/pt-BR.json

```diff
@@ -75,7 +76,16 @@
         "importWallet": "Importar carteira existente",
         "importWalletNote": "Restaure sua carteira usando a frase de recuperação",
         "addReadOnlyWallet": "Adicionar carteira somente leitura",
-        "addReadOnlyWalletNote": "Monitore ativos sem precisar usar as chaves de gasto"
+        "addReadOnlyWalletNote": "Monitore ativos sem precisar usar as chaves de gasto",
+        "sharedWallet": "Carteira Compartilhada",
+        "sharedWalletNote": "Carteira Multisig Bitcoin"
```

Comment:
```suggestion
        "sharedWalletNote": "Carteira compartilhada Bitcoin"
```

Reply:
@imstar15 Please approve.

Reply:
The meaning of this translated text is not consistent with the text of the design draft, so I kept the original meaning.

@bitcoinuser

Reply:
Hi @imstar15 and @chrisli30  please do not revert it. It's like: (Requires two or more participants to approve transactions). 

Thanks.

Reply:
@bitcoinuser Ive thought about that but I don't agree. We should notice use about this being a multi-sig wallet in description because otherwise there's no mentioning it in the page.

Reply:
Ok, changed. Please @imstar15 approve. Thanks.